src/bp/ui-studio/src/web/components/Content/Select/style.scss

```diff
@@ -10,3 +10,17 @@
   border-radius: 0;
   margin-left: -1px;
 }
+
+.actionBtn {
```

Comment:
is this style necessary for a bootstrap native feature ? inputgroup.append ? https://react-bootstrap.github.io/components/input-group/#input-group-buttons

Probably  different version of react-bootstrap I imagine

Reply:
Yep, doc is for bootstrap 4. I actually want to override the default styling to make the whole zone clickable